CMakeLists.txt

```diff
@@ -1,3 +1,12 @@
+function(testFileExists fileName)
+
+   if (NOT EXISTS ${fileName})
+      message(FATAL_ERROR "Could not find file " ${fileName})
```

Comment:
I decided create a function to test each file. Because after the second include, the cmake create a big number of messages without stop the process, and become very difficult to find out where exactly are the error message.

So, this function create a standard message, that could have more details creating a new parameter, and exit as a fatal error.